src/pages/offer/offer.tsx

```diff
@@ -1,90 +1,88 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import CommentForm from '../../components/commentForm/commentForm';
 import OfferInsideList from '../../components/offerInsideList/offerInsideList';
 import OfferGallery from '../../components/offerGallery/offerGallery';
-import ReviewsList from '../../components/reviewsList/reviewsList';
+import Comments from '../../components/comments/comments.tsx';
 import Header from '../../components/header/header';
 import Cards from '../../components/cards/cards';
-import { TComment } from '../../types/types';
+import { TComment, TOffer, TOffers, TCommentForm } from '../../types/types';
+import { TOfferGalleryChildren } from '../../components/offerGallery/offerGallery';
 import Map from '../../components/map/map';
 import { useOffers } from '../../store/selectors';
 import Loading from '../../components/loading/loading.tsx';
+import {useNavigate, useParams } from 'react-router-dom';
+import axios from 'axios';
 
-const offerGoods = [
-  'Wi-Fi',
-  'Washing machine',
-  'Towels',
-  'Heating',
-  'Coffee machine',
-  'Baby seat',
-  'Kitchen',
-  'Dishwasher',
-  'Cabel TV',
-  'Fridge'
-];
-
-export default function Offer(): JSX.Element {
+export default function Offer() {
+  const navigate = useNavigate();
+  const [offer, setOffer] = useState<TOffer | null>(null);
+  const [nearOffer, setNearOffer] = useState<TOffers[] | null>(null);
```

Comment:
если nearOffer массив то нужно называть во множественном числе
```suggestion
  const [nearOffers, setNearOffers] = useState<TOffers[] | null>(null);
```